### ISSUE TITLE
Versioning for documentation to match daemon version

### ISSUE THREAD

**Issue by lyoshenka** [lbryio/lbry-sdk#1130]
Right now the docs correspond to what's on `master`. We need docs for each daemon release so developers can see what's available in the version they are using.

https://github.com/jimporter/mike may be helpful.

**Comment by alyssaoc** [comment]
@lyoshenka Is this something we want to do?

**Comment by lyoshenka** [comment]
yes. i forgot about this when we were talking about lbry.tech docs. it shouldn't block getting the docs up because we're still pre 1.0, but it will become more important as the project matures. @eukreign @NetOperatorWibby keep this in mind when designing the docs system, but don't worry about it too much yet.

**Comment by moki1202** [comment]
hello @lyoshenka I would love to work on this. I am still new to open source though, so I would require some help. let me know if I can work on this.

**Comment by lyoshenka** [comment]
@moki1202 absolutely! happy to help. let me know if you get stuck.
